### DISCUSSION TITLE
No more data after firmware update

### DISCUSSION THREAD

**Discussion by GNCLORD-MDB** [tbnobody/OpenDTU#2403] [Q&A]
### What happened?

I have installed the firmware update v24.11.7.
Since then I have not received any more data from the inverter.
The two inverters were recognized correctly.
Everything worked before, what is the problem now?

Bin file: opendtu-generic_esp32.bin

![DTU1](https://github.com/user-attachments/assets/78fb6d49-bbcd-4551-b97c-727e96f60d4c)
![dtu2](https://github.com/user-attachments/assets/7f6b8be1-0b0c-481c-ab57-f43cc4de5a7c)
![dtu3](https://github.com/user-attachments/assets/945e8f90-9542-46fe-bd52-381cc49bff73)

### To Reproduce Bug

Install fireware and go to live view

### Expected Behavior

no data received from the inverter

### Install Method

Pre-Compiled binary from GitHub releases

### What git-hash/version of OpenDTU?

v24.11.7

### What firmware variant (PIO Environment) are you using?

generic-esp32

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of OpenDTU, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.
- [X] I have double checked that my inverter does not contain a W in the model name (like HMS-xxxW) as they are not supported.

**Comment by tbnobody** [comment]
Moving this to the discussions as it's not a bug. You just did  not apply a pin_mapping or selected the right one.

**Reply by GNCLORD-MDB** [reply]
OK, where do I get the pin mapping?

**Reply by ghost** [reply]
[Documentation](https://tbnobody.github.io/OpenDTU-docs/) -> [Firmware](https://www.opendtu.solar/firmware/) -> [Device Profiles](https://www.opendtu.solar/firmware/device_profiles/)
" Ready-to-use device profiles can be found [in the code repository](https://github.com/tbnobody/OpenDTU/tree/master/docs/DeviceProfiles). "

**Reply by GNCLORD-MDB** [reply]
105 / 5.000
ok, but that doesn't really help me, which one should I use? I have this device as a AllianceApps OpenDTU

![51rlBnClYDL _AC_SL1500_](https://github.com/user-attachments/assets/a7799e81-de8b-4c69-9b6a-37d7cbd12339)

**Comment by ghost** [comment]
With Display?

**Reply by GNCLORD-MDB** [reply]
no without

**Comment by ghost** [comment]
z.B. [nodemcu_esp32.json](https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles/nodemcu_esp32.json)
Die unter Konfigurationsverwaltung - Wiederherstellung auf den ESP hochladen. wichtig hierbei, von "Main Config" auf "Pin Mapping" umstellen.
Dann unter Hardware als Ausgewähltes Profil "CMT" einstellen und ganz unten speichern.

**Reply by ghost** [reply]
Falls nicht bekannt:
Es könnte sein, dass du die Wechselrichter heute trotzdem nicht (mehr) erreichst, das es mittlerweile dunkel ist und deine Wechselrichter mangels Spannung von den Modulen aus sind.

**Reply by GNCLORD-MDB** [reply]
so ich habe das nun gemacht und es hat sich etwas geändert in dern System Ansicht
ist das nun dann so richtig ?
![dtu2a](https://github.com/user-attachments/assets/c4af6669-dfa8-4880-9f42-f0c4f59d8d6f)

**Reply by ghost** [reply]
Nach deinem Foto bin ich wegen der Bauform / Größe davon ausgegangen, dass das Funkmodul ein cmt2300A ist.
Das hast du eben auch so eingestellt und daher steht es als konfiguriert da.
Scheint aber nicht erkannt zu werden.
Es könnte aber auch ein NRF24 sein.
Was hast du für Wechselrichter?
Bzw. stelle als Ausgewähltes Profil mal  bitte "NRF24" ein:

**Reply by GNCLORD-MDB** [reply]
ok ich habe das auf NRF24 umgestellt, geändert hat sich da nichts 

ich habe mal ein bild von der Platine gemacht 

![20241110_171307](https://github.com/user-attachments/assets/fee0fc25-99a8-4e2c-86b6-b2791d71c351)

**Reply by GNCLORD-MDB** [reply]
So nun  mal eine kompletten Neustart gemacht und nun steh da dann das bei Auswahl NRF24

![dtu2b](https://github.com/user-attachments/assets/90ae22d7-24ed-420c-a9da-26ad855a78af)

**Reply by ghost** [reply]
Google sagt, das ML01DP5 ein Funkmodul mit NRF24L01+ sein soll.
Dann wäre "NRF" als Profil richtig.
Es sollte jetzt bei dir der NRF als konfiguriert, verbunden und als Typ NRF24L01+ stehen, sowie alles grün sein.

**Reply by ghost** [reply]
Perfekt!

**Comment by GNCLORD-MDB** [comment]
Sehr schön :-)

Ich habe nun auch Daten bei Funkstatistik 

das steht nun 
![dtu1a](https://github.com/user-attachments/assets/2199f539-65c0-4c78-986d-a747f1bdf71f)

**Reply by ghost** [reply]
Es kommt erstmal nichts. Das wird sich hoffentlich morgen noch ändern.
Momentan sind deine Wechselrichter nicht erreichbar, daher 100% Fehler: nichts empfangen.
Die benötigen Spannung von den  Solarmodulen um einzuschalten, daher wird es erst morgen nach Sonnenaufgang wieder interessant.
Sollte aber klappen.

**Reply by GNCLORD-MDB** [reply]
ok dann schaue ich Morgen nochmal nach.
Danke für die Hilfe 👍
